cmd/fleet/apple_mdm.go

```diff
@@ -1,5 +1,6 @@
 package main
 
+/*
```

Comment:
Hm, not sure I understand why we had to comment out so much, my only concern is how fast we will be re-adding things, because we are using this in dogfood.

Reply:
@chiiph gotcha, I started commenting it out not knowing the extent of the existing code (and not knowing precisely how it would impact the new code), but if that's still something that needs to be used for some iterations (likely) before being completely replaced, I'll change my approach. In this case I think I'll try to keep it all as-is and just add the new features as separate things, and we can clean all of this when we're ready to make the change to the new "official" implementation.

The concern was (and will be) that this is very confusing in the meantime to know what is the old prototype code and the new official one.

Reply:
All of this code is currently unstable and subject to change until it's released. I think it would be better to remove the old code. The only issue is updating dogfood before the next deploy to reference the new configuration.

Reply:
@michalnicp Since the new mdm code will not be ready until a few releases, we decided to keep the prototype code around so that dogfood can keep using it in the meantime, and adjust it as needed when new code is ready (https://fleetdm.slack.com/archives/C03PK8PJDDE/p1669044744856369?thread_ts=1669044455.966189&cid=C03PK8PJDDE and subsequent comments).

I think for this PR what I'll do is make the prototype code use the new config flags instead of the old ones, but I'm waiting to see what we'll do regarding path flags vs raw bytes flags.